db/migrate/20171114150937_cleanup_installed_packages.rb

```diff
@@ -0,0 +1,55 @@
+class CleanupInstalledPackages < ActiveRecord::Migration
+  def up
+    create_table "katello_installed_packages_new", force: :cascade do |t|
+      t.string "name", limit: 255, null: false
+      t.string "nvra", limit: 255, null: false
```

Comment:
How about adding a unique index to this column to speed up the search

Reply:
I'm not sure that it would help, but i can time it and see

Reply:
tested and with the index it took about ~4 minutes more.